src/components/Articles.jsx

```diff
@@ -1,10 +1,38 @@
-import { fetchArticles } from "../../api"
-
-export default function Articles(){
-    fetchArticles().then((response) => {console.log(response)})
-    return (
-        <>
-        <p>Articles</p>
-        </>
-    )
-}
\ No newline at end of file
+import { useState } from "react";
+import { fetchArticles } from "../../api";
+import { Link } from "react-router-dom";
+export default function Articles() {
+  const [articles, setArticles] = useState([]);
+  fetchArticles().then((response) => {
```

Comment:
lets change this as we want to fetch articles when the page loads but this could take more time that we would like